.travis.yml

```diff
@@ -1,74 +0,0 @@
-dist: "focal"  # https://docs.travis-ci.com/user/reference/overview/
-language: python
-python:  # https://devguide.python.org/#status-of-python-branches
-  - "3.9"
-  - "3.8"
-  - "3.7"
-  - "3.6"
-  #- "nightly"
-
-# Enable pip cache: https://docs.travis-ci.com/user/caching/
-cache: pip
-env:  # https://www.djangoproject.com/download/
-  global:
-    - DJANGO_CA_SECRET_KEY=dummy
-  jobs:
-    - DJANGO=3.1.5 CRYPTOGRAPHY=3.3.1
-    - DJANGO=3.1.5 CRYPTOGRAPHY=3.2.1
-    - DJANGO=3.1.5 CRYPTOGRAPHY=3.1
-    - DJANGO=3.1.5 CRYPTOGRAPHY=3.0
-    - DJANGO=2.2.17 CRYPTOGRAPHY=3.3.1
-    - DJANGO=2.2.17 CRYPTOGRAPHY=3.2.1
-    - DJANGO=2.2.17 CRYPTOGRAPHY=3.1
-    - DJANGO=2.2.17 CRYPTOGRAPHY=3.0
```

Comment:
I did not migrate those associations between Django version and Cryptography version, but did not experience any hassle on GHA CI. Maybe it was only needed for Travis CI?

Reply:
Hm. Now, after reading #70, I recognize it might be important to also re-add the `cryptography`-axis to the test matrix?

Reply:
Yes, we DEFINITELY need cryptography in the matrix! also note that django-ca sometimes behaves differently if it detects some features.

(I think that this is not the case right now, but was in the past and will be again)

Reply:
Ok, I will try to add it. Thanks.

Reply:
I added `cryptography` to the test matrix.

Do you believe it is okay to omit `cryptography==3.0.0` already? Let me know if I should add it back.